neurom/fst/tests/test_neuritefunc.py

```diff
@@ -207,6 +207,11 @@ def test_partition():
     assert_allclose(partition,
                     (1.0, 1.0))
 
+def test_partition_asymmetry():
```

Comment:
I test the higher level functionality in "test_get_features.py". Maybe this should be moved here in a future commit.